src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.split(';');
```

Comment:
The split operation is correctly used to separate the CSS declarations. However, you should avoid creating unnecessary intermediate variables like `declarations` if you are not going to use them elsewhere. You can chain the `split` and `forEach` directly.